assets/scripts/app/load_resources.js

```diff
@@ -4,7 +4,9 @@
  * Loads images, etc and tracks progress. (WIP)
  * TODO: Rely on Promises to resolve progress
  */
-/* global _checkIfEverythingIsLoaded, Image, XMLSerializer */
+/* global Image, XMLSerializer */
```

Comment:
Instead of declaring `global` variables that are provided by the browser API let's just use `window.Image` or `window.XMLSerializer` so it is clearer that these are not put into the global scope by Streetmix itself.